src/main/java/fi/fmi/avi/converter/ConversionHints.java

```diff
@@ -369,6 +372,18 @@ public String toString() {
         KEY_WHITESPACE_SERIALIZATION_MODE = new KeyImpl(14, "Controls message white space serialization in TAC bulletins",
                 VALUE_WHITESPACE_SERIALIZATION_MODE_TRIM, VALUE_WHITESPACE_SERIALIZATION_MODE_PASSTHROUGH);
 
+        KEY_BULLETING_HEADING = new Key(15) {
```

Comment:
I see private keys `16` and `17` in lines 336-338. This makes me wonder whether `15` has already been used once. I would avoid reuse and change this to `18`.

To avoid further confusion with upcoming key numbers, I would reorder existing initializations in ascending internal key order.

Although the internal key doesn't seem to be used anywhere, but just in case...

Reply:
Changed private key 15 -> 18 and reordered keys into ascending order.